### ISSUE TITLE
Punishment doesn't seem to trigger properly for certain blocks

### ISSUE THREAD

**Issue by kaktuskuri** [N1Ran/BlockLimiter#68]
The punishment feature doesn't seem to work for some blocks. Will provide a video and a copy of the test world.

In the video, the punishment feature only seems to work as soon as the punishment is triggered for another block

[https://streamable.com/azgl29](url)
[https://ufile.io/xp0jx48r](url) zip of the torch instance where I tested the video <-- using uploadfiles.io

**Comment by N1Ran** [comment]
can you share the BlockLimiter.cfg please

**Comment by N1Ran** [comment]
wait, that is what is supposed to happen.  Punishment is base of your interval setting for punishment, it's not instant.  You running the command triggers the punishment across server which is doing what it's supposed to do

**Comment by kaktuskuri** [comment]
Ah, I only did the punishment command to make the video quick. 
Here is a video of the punishment setting with an interval of 20 seconds. If you skip till the end, it only punishes build and repair blocks after it has punished another kind of block. It doesn't punish the build and repair blocks on its own

https://www.youtube.com/watch?v=BTCQAGtfkiY&feature=youtu.be

https://pastebin.com/2W1sh6sV <-- config

If you want to check the world I tested with, its in the file I uploaded in uploadfiles.io
